### PR TITLE
Jackie/file setup

### PR DESCRIPTION
**Pull request by jackiel12**
created webpack config file and installed dependencies...have the core file structure set up

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+console.log(
+    'WTF'
+)
+
```
Comment (mesherrera) [inline src/index.js]: Remove before merge :)

**Review by mesherrera — CHANGES_REQUESTED** [review]
Just a minor fix and then merge

File: src/index.js (hunk 1 above)
Comment (jackiel12) [reply src/index.js]: removed un necessary console log

**Review by mesherrera — APPROVED** [review]
Awesome!

File: server/server.js [hunk 2]
```diff
@@ -5,6 +5,10 @@ app.get('/', (req, res) => {
   res.send('Coming soon...')
 });
 
+// app.get('/api', (req, res) => {
```
Comment (wmagee03) [inline server/server.js:8]: Make sure this get's removed later on

File: server/server.js [hunk 3]
```diff
@@ -5,6 +5,10 @@ app.get('/', (req, res) => {
...
+//   res.send('Working? Plz')
```
Comment (dpad-warrior) [inline server/server.js:9]: Remove this part.

**Review by dpad-warrior — CHANGES_REQUESTED** [review]
Great job. Just needs to remove/edit one console log.

**Review by wmagee03 — APPROVED** [review]
Everything looks good to me. If webpack is working on your end, then it's good to go

**Pull request merged**